pages/api/faucet/solana.ts

```diff
@@ -0,0 +1,29 @@
+import * as web3 from "@solana/web3.js";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+const handler = async (req: NextApiRequest, res: NextApiResponse) => {
+  if (req.method !== "POST") {
+    return res.status(400).json({ message: "Please use POST method" });
+  }
+
+  const { address } = req.body;
+
+  try {
+    const connection = new web3.Connection("https://api.devnet.solana.com");
```

Comment:
branch should be based off of the solana branch at which point u can read the RPC url from the config file for it instead of hard-coding it